jsk_panda_robot/jsk_panda_accessories/README.md

```diff
@@ -0,0 +1,7 @@
+
+# Tool mounter
+Mounting any tools onto panda-hand.  Plate mounter has M3 screw wholes placed by 10mm x 10mm.
+![](https://raw.githubusercontent.com/ykawamura96/franka_demos/mech/add_adapter_parts/franka_accessories/panda_wrist/hand_adapter.png)
```

Comment:
update URL to jsk-ros-pkg?

Reply:
Oh, sorry, I overlooked this.
I'll fix this.

Reply:
Fixed